pallets/runtime/develop/src/runtime.rs

```diff
@@ -751,7 +765,7 @@ construct_runtime!(
     {
         System: frame_system::{Module, Call, Config, Storage, Event<T>},
         // Must be before session.
-        Babe: pallet_babe::{Module, Call, Storage, Config, Inherent(Timestamp)},
+        Babe: pallet_babe::{Module, Call, Storage, Config, Inherent},
```

Comment:
Shouldn't we add `ValidateUnsigned` in its definition? This will be used in off-chain context.

Reply:
Not sure what you mean by that. This diff was caused by the macro syntax having changed. Apparently this is the new way to do things.

Reply:
In rc6 definition `ValidateUnsigned` is needed https://github.com/paritytech/substrate/blob/be8bb186d87b9d2b47a2907c9b51ae1e252362c3/bin/node/runtime/src/lib.rs#L857